src/codegen/unwinding.cpp

```diff
@@ -572,7 +577,83 @@ BoxedDict* getLocals(bool only_user_visible, bool includeClosure) {
                         const auto& locs = e.locations;
 
                         assert(locs.size() == 1);
-                        uint64_t v = frame_info.readLocation(locs[0]);
+                        uint64_t v = frame_iter.readLocation(locs[0]);
+                        if ((v & 1) == 0)
+                            is_undefined.insert(p.first.substr(12));
+
+                        break;
+                    }
+                }
+            }
+
+            for (const auto& p : cf->location_map->names) {
+                if (p.first[0] == '!'
```

Comment:
Ok, I pushed a change to make these names start with a '#' which I think should be more consistent -- could you try rebasing and seeing if you can just remove the checks for these names?